src/services/comment.service.ts

```diff
@@ -42,6 +50,10 @@ class CommentService {
     });
     return getLikes?.[0].likes || [];
   }
+
+  public async likeComment({ id, userId }: any): Promise<void> {
+    await pgDataSource.createQueryBuilder().relation(CommentEntity, 'likes').of(id).add(userId);
```

Comment:
Do we want to check if the user already liked the post?

Reply:
Can you also explain to me what this query builder does?

Reply:
So the createQueryBuilder method has a relation method there it takes the Entity where the many to many unidirectional field is and looks for the name 'likes' or whatever the field name is. ".of(id)" refers to the comment_id where we are adding the userId.

Reply:
https://typeorm.io/relational-query-builder